pages/_includes/resource_block.html

```diff
@@ -0,0 +1,45 @@
+<div class="grid-row grid-gap display-flex margin-top-2 padding-bottom-2 padding-top-2">
+
+  <div class="desktop:grid-col-8">
+    {% if resource.collection == "summaries" %}
+      <a href="{{ resource.link }}" target="_blank">
+        <h2 id="{{ resource.resource_name | slugify }}" class="margin-top-0">{{ resource.resource_name }}<i class="fa fa-xs fa-external-link-alt display-flex flex-align-center"></i></h2>
+      </a>
+    {% else %}
+      <a href="{{ "/resources/" | append: resource.slug | relative_url }}">
```

Comment:
As far as I can tell, slug is pretty much always the same as the filename. I'm pretty sure that Jekyll collections expose something very similar, like `resource.path` or `resource.url` that will get you this without having to define a `slug` in every resource. It might require some collection overrides for permalink, it's been a while since I've  done it.

Hmm, I could also be confusing it with `{% link <filename %}`. Either way, I'm pretty sure you can avoid having the duplicate `slug` front matter.

Reply:
Yes, you are correct -- `slug` is automatically generated from the `title` if the slug is not already there in the collection. That being said, we still wanted the option for people to override the slug if needed -- and when we added this `slug` option in Netlify, we were forced to add a slug for each resource (otherwise saving would result in a blank slug). This behavior doesn't hurt anything, it just makes the data more explicit (not necessarily a bad thing imo).

Good catch on `resource.url` -- making this change.

Reply:
Usually I prefer the directory structure to match the site structure, so it's obvious that if you want to change slug, change the directory/filename.

But if this is editable in the Netlify admin then that sounds good to me.

Reply:
Sounds like this is okay for now!